CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-## 2.18.0-rc.1 / 2020-04-30
+## 2.18.0 / 2020-05-05
 
 * [CHANGE] Federation: Only use local TSDB for federation (ignore remote read). #7077
```

Comment:
This seems to link to the wrong PR? Perhaps it should have been #7096?

Reply:
Nicely spotted! 

Fix: https://github.com/prometheus/prometheus/pull/7205